iphone/Classes/TiUITabProxy.m

```diff
@@ -255,6 +255,11 @@ - (void)openWindow:(NSArray *)args
   [window setIsManaged:YES];
   [window setTab:self];
   [window setParentOrientationController:self];
+
+  TiUIView *view = [window view];
+  TiViewController *controller = (TiViewController *)[window hostingController];
+  [view setFrame:controller.view.bounds];
```

Comment:
How about `[TiUtils setView:controller.view positionRect:controller.view.bounds]`? It also respects the center, although it's not as important here.